packages/swap/src/utils/statechain.ts

```diff
@@ -29,15 +32,23 @@ export const getSwapRateV3 = async ({
   destAsset,
   depositAmount,
   limitOrders,
-  dcaParams,
+  dcaParams: _dcaParams,
+  ccmParams: _ccmParams,
   excludeFees,
   brokerCommissionBps,
 }: SwapRateArgs) => {
   const client = initializeClient();
-  const dcaParameters = dcaParams
+  const dcaParams = _dcaParams
+    ? {
+        number_of_chunks: _dcaParams.numberOfChunks,
+        chunk_interval: _dcaParams.chunkIntervalBlocks,
+      }
+    : undefined;
+  const ccmParams = _ccmParams
     ? {
-        number_of_chunks: dcaParams.numberOfChunks,
-        chunk_interval: dcaParams.chunkIntervalBlocks,
+        // TODO: remove cast once https://github.com/chainflip-io/chainflip-product-toolkit/pull/325 is merged
```

Comment:
will update package in a follow up bc the changed types will require some more changes